committee-code-of-conduct/election.md

```diff
@@ -0,0 +1,78 @@
+# Kubernetes Code of Conduct Committee Elections
+
+This document outlines the process, for Code of Conduct Committee (CoCC) elections.
+
+### Length of Terms
+
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed for 2 year terms and the other 2 members will be appointed for a 1 year term.
+* In all subsequent elections the term will be for 2 years.
+* In alternating years either 2 or 3 seats will be up for election as terms expire.
+
+### Eligibility for voting
+
+* Voting will be done by members of the steering committee.
+
+### Eligibility for candidacy
+
+* Candidate commits to putting the interests of the community above the interests of their employer for all Code of Conduct Committee activities.
+* Is generally a responsible human
+* Does not have to be part of the Kubernetes or CNCF community
+* Characteristics which would be nice in any nominations:
+  * Previous experience on an Ethics Committee or Code of Conduct Committee is appreciated
+  * Has demonstrated integrity, professionalism, and positive influence within the community is appreciated
+  * Experience with the tools which we use to communicate (Zoom, Slack, GitHub, etc.) within the Kubernetes community is appreciated
+
+### Election process
+
+The steering committee will announce a request for nominations some time in July from any member of the following:
+* steering committee
+* sig chair
+* sig lead
```

Comment:
Can we change `sig lead` to `sig tech lead` for more clarity?
Should current CoCC members be eligible to nominate people?

Reply:
fixed.